tests/test_agent_service.py

```
@@ -259,6 +260,50 @@ async def _fake_run_async(*args, **kwargs):
     assert "MODE: agenda" in captured["prompt"]
 
 
+@pytest.mark.asyncio
+async def test_run_outfit_agent_emits_timeline_payload():
+    from server.services.agent_service import run_outfit_agent
+
+    emitted: list[dict] = []
+
+    async def _fake_run_async(*args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_fake_run_async`

(ANN202)